clients/gitlabrepo/client.go

```diff
@@ -285,14 +302,3 @@ func CreateGitlabClientWithToken(ctx context.Context, token string, repo clients
 func CreateOssFuzzRepoClient(ctx context.Context, logger *log.Logger) (clients.RepoClient, error) {
 	return nil, fmt.Errorf("%w, oss fuzz currently only supported for github repos", clients.ErrUnsupportedFeature)
 }
-
-// DetectGitLab: check whether the repoURI is a GitLab URI
-// Makes HTTP request to GitLab API.
-func DetectGitLab(repoURI string) bool {
```

Comment:
regarding the comment in `checker/client.go`, does it make sense to have some sort of gitlab detection still?

Reply:
I think there's a way to do without detection. `MakeGitlabRepo` and `DetectGitlab` both called `parse` and `isvalid`, so it didn't feel like they needed to be separate functions.